templates/hooks/cluster/sdk_file_end.go.tpl

```diff
@@ -0,0 +1,26 @@
+{{ $CRD := .CRD }}
+{{ $SDKAPI := .SDKAPI }}
+
+{{- $updateClusterVersion := (index $SDKAPI.API.Operations "UpdateClusterConfig") -}}
+
+{{/* Find the structure field within the operation */}}
+{{- range $memberRefName, $memberRef := $updateClusterVersion.InputRef.Shape.MemberRefs -}}
```

Comment:
I see what you are trying to do here. Applause for the innovative use of Go templating :)

I think, however, you could iterate over the `$CRD.SpecFields` map instead and that would mean you wouldn't need to dig into the `$SDKAPI.API.Operations` stuff above and you could use the `Field.Names` attribute instead of relying on the `ToNames` template function you added in aws-controllers-k8s/code-generator#256.